src/install/java/license.mdx

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
I'm not opposed either way, but why not follow the normal pattern of linking out to the license key doc here?

Reply:
Good question! The main reason is that I think our license key doc sort of muddies the waters a bit. And also I'd rather someone just went and did the thing than read a doc about it. But if you think it's worth linking to, I'm happy to.

Reply:
Generally links into the UI are a good conversion spot, so I think I'd prefer a direct link over a link to the (somewhat complicated) license key doc

Reply:
I've come up with a both/and solution here. We can always remove the link to the license key doc later if we want to.